packages/webapp/src/components/Modals/QuickAssignModal/index.jsx

```diff
@@ -19,13 +19,14 @@ export default function TaskQuickAssignModal({
 }) {
   const { t } = useTranslation();
   const selfOption = { label: `${user.first_name} ${user.last_name}`, value: user.user_id };
-  const unAssignedOption = { label: t('TASK.UNASSIGNED'), value: null };
+  const unAssignedOption = { label: t('TASK.UNASSIGNED'), value: null, isDisabled: false };
   const options = useMemo(() => {
     if (user.is_admin) {
       const options = users.map(({ first_name, last_name, user_id }) => ({
         label: `${first_name} ${last_name}`,
         value: user_id,
       }));
+      unAssignedOption.isDisabled = !isAssigned;
```

Comment:
I think disabling the ability to select unassigned is a great idea, but is there any way we can visually indicate that this option is disabled? Currently when I clone this locally and then try and assign an unassigned task, I can still see "Unassigned" in the dropdown, but can't click on it. I feel like this can lead to a confusing user experience

Reply:
Yeah, I was thinking the same, I just wasn't sure what an option should look like when its disabled. Probably just make the text gray, but let's see.

Reply:
I think this should be fine?

<img width="1194" alt="Screen Shot 2022-05-05 at 4 33 05 PM" src="https://user-images.githubusercontent.com/104768598/167021034-51f6ab09-ceaa-4919-be91-d968658152d5.png">

Reply:
I think this is good. However, I have found a place in the app that entirely removes values from the select list when they are not a valid choice. There's a chance we'll be asked to rework this for consistency. Sorry I didn't have that info for you earlier.